content/Hardware Support/UNO/Restore-the-USB-connectivity-firmware-on-UNO-R4-WiFi-with-espflash.md

```diff
@@ -36,8 +36,9 @@ Follow these steps:
 1. Run the command:
    * **Windows:** `bin\espflash write-bin -b 115200 0x0 (Get-Item .\firmware\UNOR4-WIFI-S3-*.bin).FullName`
    * **macOS/Linux:** `./bin/espflash write-bin -b 115200 0x0 firmware/UNOR4-WIFI-S3-*.bin`
-     > [!NOTE]
-     > You may need to adjust your system's security settings to allow this command if you encounter a warning pop-up. To proceed, go to _System Preferences > Privacy & Security_, and under _Security_, click _Allow Anyway_ to allow espflash to run.
+
+> [!NOTE]
+> You may need to adjust your system's security settings to allow this command if you encounter a warning pop-up. To proceed, go to _System Preferences > Privacy & Security_, and under _Security_, click _Allow Anyway_ to allow espflash to run.
```

Comment:
```suggestion
   > [!NOTE]
   > If you're using macOS, you may need to adjust your system's security settings to allow this command if you encounter a warning pop-up. To proceed, go to _System Settings > Privacy & Security_, and under _Security_, click _Allow Anyway_ to allow espflash to run.
```

Hi @jsnkan and thanks for this contribution!

This suggestion clarifies that the instructions are for macOS users, and uses the new "System Settings" name which was introduces in macOS Venture. It also indents the note to avoid breaking up the list (which resets the numbering).